internal/output/config_outputter.go

```diff
@@ -0,0 +1,50 @@
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// configPlaintextOutputFn converts the resource to plain text specifically for data from the
+// config file.
+var configPlaintextOutputFn = func(r resource) string {
+	keys := make([]string, 0)
+	for k := range r {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	lst := make([]string, 0)
+	for _, k := range keys {
+		lst = append(lst, fmt.Sprintf("%s: %s", k, r[k]))
+	}
+
+	return strings.Join(lst, "\n")
+}
+
+type configOutputterFn struct {
+	input []byte
+}
+
+// New unmarshals a single config resource and wires up a particular plain text output function.
+func (o configOutputterFn) New() (Outputter, error) {
```

Comment:
By making the output API simpler for the caller, we're forced to implement a new `New()` method in addition to a new `PlaintextOutputFn` type. If we find the tradeoff not worth it, we can reconsider.